ngraph_bridge/ngraph_encapsulate_impl.cc

```diff
@@ -134,32 +134,37 @@ Status NGraphEncapsulateImpl::GetNgExecutable(
     MemoryProfile(vm0, rss0);
 
     NGRAPH_VLOG(1) << "Compilation cache miss: " << m_name;
+    string serialized_ng_func;
     if (!m_do_aot) {
       TF_RETURN_IF_ERROR(Builder::TranslateGraph(input_shapes, static_input_map,
                                                  &m_graph, ng_function));
       ng_function->set_friendly_name(m_name);
+      serialized_ng_func = ngraph::serialize(ng_function, 4);
     } else {
       auto itr = m_aot_functions.find(signature);
       if (itr == m_aot_functions.end()) {
         return errors::Internal(
             "Expected to find AOT precompiled ng function of signature: ",
             signature);
       }
-      ng_function = ng::deserialize(itr->second);
+      serialized_ng_func = itr->second;
     }
 
-    auto function_size = ng_function->get_graph_size() / 1024;  // kb unit
+    int function_size;
+    if (!m_do_aot) {
+      function_size = ng_function->get_graph_size() / 1024;  // kb unit
+    }
```

Comment:
If not not - then function size not calculated - is that ok?

Reply:
Here is the description: https://github.com/NervanaSystems/ngraph/blob/0a6f5bca7de2013518fe5124534b6a6ca97970f7/src/ngraph/function.hpp#L109

Given that the function is not a reliable measure, will remove it as discussed f2f.

Currently we specify cache depth rather than memory consumption through the environment flag.